README.md

```diff
@@ -1,135 +1,21 @@
-[![Travis Build Status](https://img.shields.io/travis/rust-lang/rust.svg?style=flat-square)](https://travis-ci.org/fabric8-ui/fabric8-ux)
 [![License](https://img.shields.io/github/license/mashape/apistatus.svg?style=flat-square)](https://opensource.org/licenses/Apache-2.0)
 
-[![Platform Designs](https://img.shields.io/website-up-down-green-red/http/shields.io.svg?label=Designs:Platform&style=flat-square)](https://fabric8-ui.github.io/fabric8-ux/designs/platform)
-[![Planner Designs](https://img.shields.io/website-up-down-green-red/http/shields.io.svg?label=Designs:Planner&style=flat-square)](https://fabric8-ui.github.io/fabric8-ux/designs/planner)
-
 # User Experience Repo
 
 Design Stories, Documents and UX Mockups related to the [fabric8-ui](https://github.com/fabric8-ui/fabric8-ui) and [fabric8-planner](https://github.com/fabric8-ui/fabric8-planner) repositories.
```

Comment:
Should this really say "Design stories, documents, and UX mockups related to the Fabric8 organization and OpenShift.io organization"?

Reply:
nope, I can remove it.